### PR TITLE
Add -p and -a options for -r

### PR DESCRIPTION
**Pull request by brunobell**
add -p, --player to read with your own players, festival for offline reading and mpg123 or sox for online reading, default to festival,
add add -a --accent to read online with your accent: uk or us, default to auto.

😄  now you can try this:
`python ydcv.py hello -r -p mpg123 -a uk`

### CODE REVIEW

File: ydcv.py [hunk 1]
```diff
@@ -153,14 +158,39 @@ def print_explanation(data, options):
...
+                        else:
+                            accent_file = '/tmp/ydcv-' + str(uuid1()) + '.mp3'
+                            with open(accent_file, 'wb') as f:
+                                f.write(urlopen(accent_url).read())
+                            if options.player == 'mpg123':
+                                Popen('mpg123 -q ' + accent_file, shell=True)
+                            elif options.player == 'sox':
+                                Popen('play -q ' + accent_file, shell=True)
+                            try:
+                                Popen('sleep 5 && rm ' + accent_file, shell=True)
```
Comment (felixonmars) [inline ydcv.py]: It would be better to wait until the process end instead of a fixed 5 seconds.

**Comment by brunobell** [comment]
Ok, I've committed some tiny fix, please check it.

**Comment by felixonmars** [comment]
Thanks. Please resolve the conflicts and I'll merge.

**Comment by brunobell** [comment]
@felixonmars Sure, I've done rebase :)

**Pull request merged**